doc/api/process.md

```diff
@@ -529,7 +528,7 @@ running the `./configure` script.
 
 An example of the possible output looks like:
 
-```js
+```txt
```

Comment:
Ditto about `txt` language. Or, disabling ESLint for this block and keep `js` also works with me.